### ISSUE TITLE
Difficulty reading while hovering over metadata in high-contrast mode

### ISSUE THREAD

**Issue by brockfanning** [open-sdg/open-sdg#2]
When in high-contrast mode on an indicator page, users will have difficulty reading the metadata that they are hovering over. Because of the yellow hover-background, it will be either white text on yellow background, or worse, yellow links on yellow background.

**Comment by AnnCorp** [comment]
When not in high-contract mode, labels are in bold black, the variable info is plain black, hyperlinks are plain blue and hover over keeps all colours the same but uses a grey background.
In high contrast when not hovering over metadata, labels are bold white and  text is plain white, hyperlinks are yellow and the background is black.
In high contrast when hovering over metadata, the labels are in bold white, the variable text is black, hyperlinks are yellow text  and the background is yellow so its difficult to read the labels and impossible to read hyperlinks. If we changed the labels to black bold that would help that but am not sure on best way to distinguish hyperlinks - as shot-term fix, at least making them black would be better than yellow on yellow. Perhaps we need some specialist accessibility input on this?

**Comment by dougmet** [comment]
@JABedford is going to take a look for us.

**Comment by JABedford** [comment]
Just left a PR for a quick fix to this issue. #80 

As @AnnCorp mentioned, the hyperlinks are the tricky bit here. I have changed them to blue and added the default text decoration back so that hyperlinks are underlined in the hover state so at least they are slightly distinguishable.

**Comment by dougmet** [comment]
@AnnCorp I'll make a version on the test site for you to inspect

**Comment by dougmet** [comment]
usual test server address http://XX.XX.XX.XX/test-high-contrast-no-merge/5-3-1/ 
![image](https://user-images.githubusercontent.com/5878305/52420893-da9dcf80-2aea-11e9-9a03-a62088b8e7e2.png)

**Comment by dougmet** [comment]
Fixed by #80

**Comment by Norric1Admin** [comment]
This is a definite improvement :+1: 
BEFORE:
![image](https://user-images.githubusercontent.com/37659239/53338065-a0aa3700-38fa-11e9-952e-25a69a67272f.png)
AFTER:
![image](https://user-images.githubusercontent.com/37659239/53338106-c0d9f600-38fa-11e9-8142-09d2f895f8cd.png)
